src/v/config/configuration.cc

```diff
@@ -144,12 +144,32 @@ configuration::configuration()
   , coproc_max_ingest_bytes(*this, "coproc_max_ingest_bytes")
   , coproc_max_batch_size(*this, "coproc_max_batch_size")
   , coproc_offset_flush_interval_ms(*this, "coproc_offset_flush_interval_ms")
-  , enable_data_transforms(
+  , data_transforms_enabled(
       *this,
-      "enable_data_transforms",
+      "data_transforms_enabled",
       "Enables WebAssembly powered Data Transforms directly in the broker",
       {.needs_restart = needs_restart::yes, .visibility = visibility::user},
       false)
+  , data_transforms_per_core_memory_reservation(
+      *this,
+      "data_transforms_per_core_memory_reservation",
+      "The amount of memory to reserve per core for all WebAssembly Virtual "
+      "Machines. Memory is reserved on boot. The maximum number of functions "
+      "that can be deployed to a cluster is equal to "
+      "data_transforms_per_core_memory_reservation / "
+      "data_transforms_per_function_memory_limit",
+      {.needs_restart = needs_restart::yes, .visibility = visibility::tunable},
+      20_MiB)
+  , data_transforms_per_function_memory_limit(
+      *this,
+      "data_transforms_per_function_memory_limit",
+      "The amount of memory to give an instance of a WebAssembly Virtual "
+      "Machine. The maximum number of functions "
+      "that can be deployed to a cluster is equal to "
+      "data_transforms_per_core_memory_reservation / "
+      "data_transforms_per_function_memory_limit",
+      {.needs_restart = needs_restart::yes, .visibility = visibility::tunable},
+      2_MiB)
```

Comment:
not that chunked_fifo is large or anything, but even at 20/2=10 data pools you're not using >10x of the default capacity of 128 entries?

Reply:
I changed the chunk size to be 16.